treon/treon.py

```diff
@@ -2,13 +2,14 @@
 """
 Usage:
   treon
-  treon [PATH] [--threads=<number>] [-v]
+  treon [PATH] [--threads=<number>] [-v] [--exclude=<string>]...
 
 Arguments:
   PATH                File or directory path to find notebooks to test. Searches recursively for directory paths. [default: current working directory]
 
 Options:
   --threads=<number>  Number of parallel threads. Each thread processes one notebook file at a time. [default: 10]
+  -e=<string> --exclude=<string>   Option for excluding files that start with this string. Can be provided more than once.
```

Comment:
Maybe reword it to something like, 

"Option for excluding files or entire directories from testing. All files whose absolute path starts with the specified string are excluded from testing. This option can be specified more than once to exclude multiple files or directories."

Your wording is also technically correct but I just wanted to hint users that this option has capability to exclude directories as well.